experiment/builder_runner.py

```diff
@@ -591,10 +608,26 @@ def _run_with_retry_control(target_path: str, *args, **kwargs) -> bool:
 
   def build_and_run(self, generated_project: str, target_path: str,
                     iteration: int) -> tuple[BuildResult, Optional[RunResult]]:
+    """Builds and runs the fuzz target for fuzzing."""
     build_result = BuildResult()
+
     if not self._pre_build_check(target_path, build_result):
       return build_result, None
 
+    try:
+      return self.build_and_run_cloud(generated_project, target_path, iteration,
+                                      build_result)
+    except Exception as err:
+      logging.warning(
+          'Error occurred when building and running fuzz target on cloud'
+          '(attempt %d): %s', iteration, err)
+      traceback.print_exc()
```

Comment:
Should we use `format_exc()` instead and use `logging` for this?

Reply:
Done!